content/software-triage.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Software Troubleshooting Triage
+description: >
+ A troubleshooting triage to help you self-diagnose software issues.
+keywords:
+ - support
+ - software
+ - self-help
+ - tech-support
+ - troubleshooting
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: getting-help
+tableOfContents: true
+---
+
+## Gather Tools
+
+- [Backups](/articles/backup-files)
+- [Live Disk](/articles/live-disk)
+- [Logs](/articles/create-logs)
+
+## Suspend, Reboot, Shut Down
+
+Do the symptoms go away if the system is suspended, rebooted or shut down and then started up again? Sometimes repeated suspends or reboots are all that are needed to clear an issue out. It's also a good idea to test reboot and shutdown separately as they handle memory and power-management differently.
```

Comment:
Should we recommend suspend/resume cycles for troubleshooting? In my experience, suspend is usually the cause of problems, not the solution.

Reply:
Suspend and resuming will sometimes fix gdm refresh issues if the display is stuck. Shut the lid, open it again, and we're back.

Reply:
Switching TTYs seems like a better way to refresh gdm, but fair enough.